packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -516,8 +516,11 @@ export function useSelectableCollection(options: AriaSelectableCollectionOptions
       }
 
       if (modality === 'keyboard' || didAutoFocusRef.current) {
-        scrollIntoView(scrollRef.current, element);
-
+        requestAnimationFrame(() => {
```

Comment:
should this wrap the below scrollIntoViewport as well? Not sure if the order is going to matter. Might be a good idea to clean up the raf as well

Reply:
maybe it would be good to test it without and see if it needs it?

Reply:
did some quick testing and haven't found any weirdness so ok for now. We can test more during the testing session